awx/ui_next/src/screens/Inventory/Inventories.jsx

```diff
@@ -27,7 +27,7 @@ class Inventories extends Component {
     };
   }
 
-  setBreadCrumbConfig = (inventory, group) => {
+  setBreadCrumbConfig = (inventory, nestedResource) => {
```

Comment:
We are passing this function down several levels at this point.  We should probably consider thinking about refactoring this somehow.

Reply:
Yeah, we should probably look into rewriting this in the future. We could leverage the use of hooks in Breadcrumbs 2.0. 

The breadcrumbConfig below has a few lines of duplication. To clean it up, I would group the related breadcrumbs, arrange by order of appearance, and remove any duplication.